utils/tests.go

```diff
@@ -33,18 +33,28 @@ func GeneratePipelineWithEnvironment() meroxa.Pipeline {
 	return p
 }
 
-func GenerateResource() meroxa.Resource {
-	return meroxa.Resource{
+func GenerateResource(resourceName, resourceState string) meroxa.Resource {
```

Comment:
I like the versatility of including this option. 

I feel like I haven't done this in the past in this repository, so I definitely understand you taking the approach of sending empty strings in those cases we didn't care about the name of state, but maybe it'd be more idiomatic to define another function that doesn't pass name or state, that could call this one with those empty strings values.

i.e.:

Define:

```go
func GenerateResource() meroxa.Resource {
  return GenerateResourceWithNameAndStatus("", "")
}
```

And still call `GenerateResource()` when we don't mind using a specific name or state.

Reply:
That's much nicer to read indeed and I can refactor this accordingly!